src/bp/ui-studio/src/web/views/Content/UsageModal.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react'
+import { Modal } from 'react-bootstrap'
+import ReactTable from 'react-table'
+
+import { ContentUsage } from '.'
+import style from './style.scss'
+
+interface Props {
+  usage: ContentUsage[]
+  handleClose: Function
+}
+
+interface State {
+  page: number
+  pageSize: number
+}
+
+export default class UsageModal extends React.Component<Props, State> {
+  state = {
+    page: 0,
+    pageSize: 5
+  }
+
+  getTableColumns() {
+    return [
+      {
+        Header: 'Type',
+        filterable: false,
+        accessor: 'type',
+        width: 150
+      },
+      {
+        Header: 'Name',
+        filterable: false,
+        accessor: 'name',
+        width: 150
+      },
+      {
+        Header: 'Node',
+        filterable: false,
+        accessor: 'node',
+        width: 150
+      },
+      {
+        Header: 'Count',
+        filterable: false,
+        accessor: 'count',
+        width: 50
+      }
+    ]
+  }
+
+  fetchData = (state: State) => {
+    this.setState({
+      pageSize: state.pageSize,
+      page: state.page
+    })
+  }
+
+  render() {
+    const pageCount = Math.ceil(this.props.usage.length / this.state.pageSize)
+    return (
+      <Modal
```

Comment:
We are trying to get rid of bootstrap, please use Blueprint for the modal. Also, we're trying to use hooks as much as possible (less boilerplate). Here's an example: https://github.com/botpress/botpress/pull/2880/files